ruby/alias_manager.rb

```diff
@@ -0,0 +1,77 @@
+=begin
+ 	 
+-Create a method that swaps their first and last name
+-Create a method that changes all the vowels to the next vowel in 'aeiou'
+	-Create a loop that goes through all the letters of the name
+	-When the letter matches a letter in vowels list, 
+	change it to the next vowel in the vowels list
+	-Until all the letters are swapped, do the loop
+	-Save the newly swapped name in a variable
+-Create a method that changes all the rest of consonants to the next
+	-Iterate through all the letters and change the letter to the next one
+	in the consonants list.
+-Get user input for name and give the fake name as output until they say quit.
+-Print all the data user entered
+=end
+
+#Checking Felicia Torres 
+#real_name = "Felicia Torres"
+#swap_name = real_name.split(" ").reverse.join(" ")
+#swap_name = swap_name.downcase
+
+#Change vowels in name to next vowels
+def next_vowel(name)
+	vowels = ["a", "e", "i", "o", "u"]
+	index = 0
+	name = name.split('') # string to array
+	name_change1 = name.map do |letter| 
+		if vowels.include? letter 
+			vowels.rotate(1)[vowels.index(letter)]
+		else
+			letter
+		end
+	end
+	name_change1.join  # new array convert back to string
+end
+
+#Change the consonants to next consonant
+def next_consonant(name)
```

Comment:
It looks like this method is very similar to your `#next_vowel` method.  I would consider combining them.

For instance, you could have a general method like this:
```ruby
def next_letter(letter, letters)
   letters.rotate[letters.index(letter)]
end
```

And then pass in letters like this:
```ruby
name = name.split("")
changed_name = find_next_letters(name)
changed_name.join

def find_next_letters(name)
  name.map do |letter|
    if VOWELS.include?(letter)
      next_letter(letter, VOWELS)
    else 
      next_letter(letter, CONSONANTS)
    end
  end
end
```